WooCommerce/WooCommerceScreenshots/WooCommerceScreenshots.swift

```diff
@@ -43,38 +43,38 @@ class WooCommerceScreenshots: XCTestCase {
             return false
         }
 
-        try MyStoreScreen()
-
         // My Store
-        .dismissTopBannerIfNeeded()
-        .thenTakeScreenshot(named: "order-dashboard")
+        try TabNavComponent()
+            .goToMyStoreScreen()
```

Comment:
As explained in the commit e6cef861531d6934b3e9bb56420b58b21eb119fe, we need to ensure that we interact with the app before taking any screenshots, otherwise in the first screenshot, the notifications permission dialog would be still visible, in the past, this interaction used `app.tap()`, but as Hafiz found out, it was causing some issues, and he removed it in this PR https://github.com/woocommerce/woocommerce-ios/pull/14492, so to solve both issues, I updated the logic to force tapping on the Dashboard bottom navigation icon.